NanostreamDataflowMain/webapp/src/main/app-vue-js/src/api.js

```diff
@@ -1,42 +1,33 @@
 import config from './config.js';
+import FetchUtil from './fetch.util.js';
 
-// const urlPrefix = "http://localhost:8080/api/v1";
-const urlPrefix = "https://" + config.firebase.projectId + ".appspot.com/api/v1";
+const urlPrefix = "http://localhost:8080/api/v1";
```

Comment:
Why do we have localhost here?

Reply:
I use it for development purpose. Forgot to restore it back. Fix: https://github.com/allenday/nanostream-dataflow/pull/179